perun-web-gui/src/main/java/cz/metacentrum/perun/webgui/tabs/memberstabs/CreateServiceMemberInVoTabItem.java

```diff
@@ -204,6 +204,20 @@ public boolean validateTextBox() {
 						return false;
 					}
 
+					// check on login in password if login is longer than 2 chars
+					// TODO - check for name/surname too
+					String pass = serviceUserPassword.getTextBox().getValue();
+					String login = serviceUserLogin.getTextBox().getValue();
+					if (login.length() > 2) {
+						if (pass.toLowerCase().contains(login.toLowerCase()) ||
+								pass.toLowerCase().contains(Utils.reverseString(login.toLowerCase())) ||
```

Comment:
Do you need to perform these checks? Wouldn't the following conditions handle these situations?

Reply:
You are right. There is no point in duplicate check.